copytoclipboard2.js

```diff
@@ -138,7 +158,7 @@ function initCopyText() {
           `https://api.spotify.com/v1/tracks/${id}`,
         );
         sendToClipboard(
-          res.name + ' by ' + res.artists.map(a => a.name).join(', '),
+          res.name + ' ' + res.artists.map(a => a.name).join(', '),
```

Comment:
This should have something to separate song name from artist names, maybe a `;`

Reply:
did that